packages/hooks/src/index.ts

```diff
@@ -64,6 +67,7 @@ export {
   Student,
   Trainings,
   User,
+  getResetApiCache,
```

Comment:
I would think, for consistency sake, we don't export this method.. and it is used by the code by way of `Admin.getResetApiCache` .. I was thinking along the lines of we export all `use*` hooks individually as a convenience, and everything else in the namespace of the api it relates to `Admin.xxxxxxxx` .. what do you think?

Reply:
Sounds good